host/bcm283x/gpio.go

```diff
@@ -325,13 +328,18 @@ func (p *Pin) FastOut(l gpio.Level) {
 //
```

Comment:
It is very important in the documentation to call out that `Halt()` must be called before exiting the process or the host will be in broken state, potentially requiring an host hard reboot.

Reply:
Agree, especially DMA is problematic because it runs a infinite loop and ownership is not clear. I am not familiar with godoc but can I assume if I update the comments, godoc is updated automatically?

Reply:
godoc.org is updated when you click the refresh link at the bottom. You can run godoc locally with:

go get -u -v golang.org/x/tools/cmd/godoc
godoc -http=:6060
Then visit http://localhost:6060/pkg/periph.io/x/periph/host/bcm283x

Reply:
Thanks for the explanation!